src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/AbstractGenerateEqualsAndGetHashCodeService.cs

```diff
@@ -33,7 +33,7 @@ public async Task<Document> FormatDocumentAsync(Document document, SyntaxFormatt
 
         var formattedDocument = await Formatter.FormatAsync(
             document, s_specializedFormattingAnnotation,
-            options, rules, cancellationToken).ConfigureAwait(false);
+            options, rules.ToImmutableArray(), cancellationToken).ConfigureAwait(false);
```

Comment:
>ToImmutableArray [](http://example.com/codeflow?start=27&length=16)

nit: not real need for the list allocation above

Reply:
done.